Cargo.toml

```diff
@@ -104,7 +104,7 @@ hyper-tls = "0.5"
 into_other = { path = "components/into_other", default-features = false }
 itertools = "0.10"
 keys = { path = "components/keys", default-features = false }
-kvproto = { git = "https://github.com/pingcap/kvproto.git" }
```

Comment:
You can use the `patch` section ([here](https://github.com/tikv/tikv/blob/e278777f97cc71aa60cf21550c5e8f55bfa95b84/Cargo.toml#L208-L210) is the template) so that you can avoid changing every Cargo.toml in the sub modules

Reply:
Ah! cool